backend/app/static/js/script.js

```diff
@@ -76,10 +76,12 @@ $(document).on("ready", function () {
 
     $("#picker").change(function(){
         $("#preview-btn").prop("disabled",false);
+	$("#preview").css("background","");
     });
 
     $("input[name=img-default]").change(function(){
         $("#preview-btn").prop("disabled",false);
+	$("#picker").val("");
```

Comment:
This is not the file to be changed. This is to be removed file. Change the front-end file instead.

Reply:
I changed the file in backend, scripts are not available in frontend.

Reply:
@sagar-kalra pls see this file: https://github.com/fossasia/badgeyay/blob/development/frontend/public/js/script.js